packages/helpers/src/components/Portal.js

```diff
@@ -11,11 +11,16 @@ const propTypes = {
 const defaultProps = {
   children: null,
   active: true,
-  container: document.body,
+  container: null,
 };
 
-const Portal = ({ children, active, container }) =>
-  active ? ReactDOM.createPortal(children, container) : <React.Fragment>{children}</React.Fragment>;
+function Portal({ children, active, container }) {
+  return active ? (
+    ReactDOM.createPortal(children, container || document.body)
```

Comment:
@DanilAgafonov, this change is needed as we are having an issue in projects where the container is null which then throws a JS error in the app.

cc: @AndreyChernykh

Reply:
Good 🎣 @oscarkwan

Reply:
@DanilAgafonov yes, it looks like in some cases when `defaultProps` are defined, `document.body` does not exist yet.

Reply:
@AndreyChernykh since `defaultProps` are defined at the top level in the module, they will be evaluated when the bundle script is loaded. Since in projects the bundle script element is in the `head`, `document.body` will not exist yet.